scipy/special/meson.build

```diff
@@ -134,6 +134,16 @@ cephes_lib = static_library('cephes',
   dependencies: [py3_dep, np_dep],
 )
 
+py3.extension_module('_special_ufuncs',
+  ['_special_ufuncs.cpp', '_special_ufuncs_docs.cpp', 'sf_error.cc'],
+  c_args: use_math_defines,
```

Comment:
This should be `cpp_args` for C++ code. It looks like it can be deleted though; it's only needed when Cython is involved.